.cirrus.yml

```diff
@@ -71,6 +71,7 @@ task:
   timeout_in: 60m
   environment:
     OS_NAME: freebsd
+    CI_DFLAGS: -version=TARGET_FREEBSD12
```

Comment:
No idea why dmd fails to compile with `src/core/checkedint.d(691): Error: divide by zero` if `DFLAGS` is exported in the environment.